python/cugraph/cugraph/dask/common/input_utils.py

```diff
@@ -15,8 +15,8 @@
 
 from collections.abc import Sequence
 from collections import OrderedDict
-from dask_cudf.core import DataFrame as dcDataFrame
-from dask_cudf.core import Series as daskSeries
+from dask_cudf import DataFrame as dcDataFrame
+from dask_cudf import Series as daskSeries
```

Comment:
NOTE: **All** imports from `dask_cudf.core` should be avoided, because these imports are always using "legacy" dask-cudf. Importing from the top-level `dask_cudf` module are automatically routed to the proper API. There is no way to protect against `dask_cudf.core` imports **yet**, because some query-planning logic still needs to find/use specific legacy code.